platform-sdk/swirlds-common/src/main/java/com/swirlds/common/crypto/Hash.java

```diff
@@ -79,7 +79,22 @@ public Hash(final byte[] value) {
      * 		the digest type
      */
     public Hash(final byte[] value, final DigestType digestType) {
-        this(value, digestType, false, false);
+        if (value == null) {
```

Comment:
Maybe a good time to add nullity annotations to the constructor arguments.

Reply:
added